README.md

```diff
@@ -150,6 +154,29 @@ module "my-ez-fargate-service" {
 }
 ```
 
+### Internal Example
```

Comment:
Scoot this over to the examples directory if there's some value, but lets not be adding new stuff to the README